content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -217,6 +217,11 @@ sudo systemctl enable --now kubelet
 
   - You can leave SELinux enabled if you know how to configure it but it may require settings that are not supported by kubeadm.
 
+  - If the `baseurl` fails because your Red Hat-based distribution cannot interpret `basearch`, replace `\$basearch` with your computer's architecture.
+  Type `uname -m` to see that value.
+  Supported architectures include x86_64, aarch64, armhfp, ppc64le, and s390x.
```

Comment:
i prefer if we don't enumerate the supported architectures on the kubeadm page.

Reply:
I removed that line.